csharp/Platform.Numbers/ArithmeticExtensions.cs

```diff
@@ -1,51 +1,51 @@
-﻿using System.Runtime.CompilerServices;
+using System.Runtime.CompilerServices;
 
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
 
 namespace Platform.Numbers
 {
     /// <summary>
-    /// <para>.</para>
-    /// <para>.</para>
+    /// <para>Provides a set of extension methods that perform mathematical operations on arbitrary object types.</para>
+    /// <para>Предоставляет набор методов расширения выполняющих математические операции для объектов произвольного типа.</para>
     /// </summary>
     public static class ArithmeticExtensions
     {
         /// <summary>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>Increments the variable passed as an argument by one.</para>
+        /// <para>Увеличивает переданную в качестве аргумента переменную на единицу.</para>
         /// </summary>
         /// <typeparam name="T">
         /// <para>The number type.</para>
         /// <para>Тип числа.</para>
         /// </typeparam>
         /// <param name="x">
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>Reference to the variable being incremented.</para>
+        /// <para>Ссылка на увеличиваемую переменную.</para>
         /// </param>
         /// <returns>
-        /// <para>.</para>
-        /// <para>.</para>
+        /// <para>Increased the value of the argument by one.</para>
```

Comment:
`The value of the argument increased by one.` The words order fix. Use https://app.grammarly.com/ (free version is enough) next time to find grammar errors.